### PR TITLE
Add remove_duplicate token filter docs #8273

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Add remove_duplicate token filter docs

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/8273

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas Will you review this PR for technical accuracy, or have a peer review it? Thank you.

File: _analyzers/token-filters/remove-duplicates.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+title: Remove duplicate
```
Comment (natebower) [inline _analyzers/token-filters/remove-duplicates.md:3]:
```suggestion
title: Remove duplicates
```

File: _analyzers/token-filters/remove-duplicates.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+parent: Token filters
+nav_order: 350
+---
+
+# Remove duplicate token filter
```
Comment (natebower) [inline _analyzers/token-filters/remove-duplicates.md:8]:
```suggestion
# Remove duplicates token filter
```

File: _analyzers/token-filters/remove-duplicates.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+The `remove_duplicates` token filter is used to remove duplicate tokens that are generated at the same position during analysis.
+
+## Example
+
+The following example creates an index with a `keyword_repeat` token filter. The filter adds a `keyword` version of each token at the same position as the token itself and then uses a `kstem` to create a stemmed version of the token:
```
Comment (natebower) [inline _analyzers/token-filters/remove-duplicates.md:14]:
```suggestion
The following example request creates an index with a `keyword_repeat` token filter. The filter adds a `keyword` version of each token at the same position as the token itself and then uses a `kstem` to create a stemmed version of the token:
```

File: _analyzers/token-filters/remove-duplicates.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+```json
+GET /example-index/_analyze
+{
+  "analyzer": "custom_analyzer",
+  "text": "Slower turtle"
+}
+```
+{% include copy-curl.html %}
+
+The response contains the token `turtle` twice in the same position:
```
Comment (natebower) [inline _analyzers/token-filters/remove-duplicates.md:49]: We have previously used "at" the same position (lines 10 and 14). "in" actually sounds better to me. Please make consistent.

File: _analyzers/token-filters/remove-duplicates.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+title: Remove duplicate
+parent: Token filters
+nav_order: 350
+---
+
+# Remove duplicate token filter
+
+The `remove_duplicates` token filter is used to remove duplicate tokens that are generated at the same position during analysis.
```
Comment (kolchfa-aws) [inline _analyzers/token-filters/remove-duplicates.md:10]:
```suggestion
The `remove_duplicates` token filter is used to remove duplicate tokens that are generated in the same position during analysis.
```

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**